src/app/domain/sharing/SharingService.ts

```diff
@@ -0,0 +1,59 @@
+import ReceiveSharingIntent from '@mythologi/react-native-receive-sharing-intent'
+
+import Minilog from 'cozy-minilog'
+
+const filesToUpload: Record<string, unknown>[] = []
+let isServiceInitialized = false
+
+const sharingLogger = Minilog('🗃️ SharingService')
+
+const initSharingMode = (): void => {
+  sharingLogger.info('Init sharing mode')
+
+  if (isServiceInitialized) {
+    sharingLogger.info('Service already initialized')
+    return // Exit early if the service is already initialized
```

Comment:
nit: this comment doesn't add knowledge as the `if(isServiceInitialized) return` is already self explanatory. Also the `.log()` add even more explanation. So in my opinion we can remove this comment